adr/00004-sidekiq-and-redis.md

```diff
@@ -0,0 +1,21 @@
+# 6. Sidekiq and Redis
```

Comment:
We already have an ADR that covers this https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/adr/00002-reuse-existing-architecture.md#decision

Reply:
Ah yep, I forgot about that one.